### PR TITLE
fix UX issues in move debugger

### PR DESCRIPTION
**Pull request by nicolodavis**
#### Checklist

* [x] Use a separate branch in your local repo (not `master`).
* [x] Test coverage is 100% (or you have a story for why it's ok).

Fixes https://github.com/nicolodavis/boardgame.io/issues/618

### CODE REVIEW

File: src/client/debug/main/InteractiveFunction.svelte [hunk 1]
```diff
@@ -69,10 +71,14 @@
...
 
 <div class="move" class:active on:click={Activate}>
-  {name}(<span
+  <span>{name}</span>
+  <span>(</span>
+  <span
     class="arg-field"
     bind:this={span}
     on:blur={Deactivate}
+    on:keypress|stopPropagation={() => {}}
```
Comment (delucis) [inline src/client/debug/main/InteractiveFunction.svelte:80]: First time reviewing Svelte code, but am I understanding correctly that the important change is this line preventing `on:keypress` listeners firing elsewhere while editing the contenteditable span and that that span is displayed conditionally somehow?

**Comment by delucis** [comment]
This is an additional change, but given it’s in the same scope:

When the debug panel is hidden using the <kbd>.</kbd> shortcut, the hotkeys for toggling which panel is displayed (<kbd>m</kbd>, <kbd>l</kbd>, <kbd>i</kbd> & <kbd>a</kbd>) are still active. This is a minimal bug, because the controls within the panels seem to be disabled correctly, but might be a simple fix to include here to ensure the panels aren’t toggled while hidden.

**Comment by delucis** [comment]
Did some local testing and this looks good for disabling shortcuts inside the move content editables 👍 

To fix the copy/paste issue, I think we can add checks here:

https://github.com/nicolodavis/boardgame.io/blob/06122c7e5b4e0f2f0cfe13ce8bb899c6e37903ad/src/client/debug/main/Hotkey.svelte#L26

Something like:

```js
    if (
      !$disableHotkeys && !disable &&
      !e.ctrlKey && !e.metaKey &&
      e.key == value
    ) { 
```

I don’t think we need to check for alt or shift keys as they tend to produce different `e.key` values?

To fix toggling the displayed pane while the debug panel is hidden, I think we need to move this code inside an if statement that checks if `visible` is true:

https://github.com/nicolodavis/boardgame.io/blob/06122c7e5b4e0f2f0cfe13ce8bb899c6e37903ad/src/client/debug/Debug.svelte#L37-L41

**Comment by nicolodavis** [comment]
Thanks @delucis!

**Pull request merged**